core/src/avm2/globals/flash/events/StageVideoEvent.as

```diff
@@ -0,0 +1,36 @@
+package flash.events
+{
+    
+    public class StageVideoEvent extends Event
+    {
+        public const codecInfo:String; 
+        public static const RENDER_STATE:String = "renderState"; // The StageVideoEvent.RENDER_STATE constant defines the value of the type property of a renderState event object.
+        public static const RENDER_STATUS_ACCELERATED:String = "accelerated"; // Deprecated since Flash Player 10.2, AIR 3: Please Use flash.media.VideoStatus.ACCELERATEDIndicates that hardware is decoding and displaying the video.
+        public static const RENDER_STATUS_SOFTWARE:String = "software"; // Deprecated since Flash Player 10.2, AIR 3: Please Use flash.media.VideoStatus.SOFTWAREIndicates that software is decoding and displaying the video.
+        public static const RENDER_STATUS_UNAVAILABLE:String = "unavailable"; // Deprecated since Flash Player 10.2, AIR 3: Please Use flash.media.VideoStatus.UNAVAILABLEIndicates that displaying the video using the StageVideo object is not possible.
```

Comment:
These three constants are also AIR-only.

Reply:
Additionally, there is a missing space in all their comments:
`... Please Use flash.media.VideoStatus.SOFTWAREIndicates that software ...`